packages/core/src/command/hl7v2-subscriptions/hl7v2-roster-generator.ts

```diff
@@ -186,6 +187,15 @@ export function genderOtherAsUnknown(gender: GenderAtBirth): GenderAtBirth {
   return gender === otherGender ? unknownGender : gender;
 }
 
+export function genderEnumerated(gender: GenderAtBirth) {
```

Comment:
This seems very tied to Louisiana, so maybe name it accordingly? Or is there a standard backing these values?